pkg/apiserver/webhooks.go

```diff
@@ -269,7 +262,11 @@ func watchCertificates(directory, certFile, keyFile string) {
 		klog.Fatalf("failed to start certificate watcher: %v", err)
 	}
 	defer watcher.Close()
-
+	// start the watcher with the dir to watch
+	err = watcher.Add(directory)
+	if err != nil {
+		klog.Errorf("Error in running watcher: %v", err)
+	}
 	// if the watcher notices any changes on the mount dir of the secret such as creations or writes to the files in this dir
 	// attempt to load tls cert using the new cert and key files provided and log output
 	done := make(chan bool)
```

Comment:
I'm not sure about keeping this channel open forever. Potential for memory leaks.

Reply:
addressed in https://github.com/nephio-project/porch/pull/53/commits/e71482d0ac353a368ae88286f8a06b8bb61cdb77